tests/cases/standalone/create/create.sql

```diff
@@ -35,3 +35,13 @@ DROP TABLE times;
 DROP TABLE test1;
 
 DROP TABLE test2;
+
+CREATE TABLE test_pk (timestamp BIGINT TIME INDEX, host STRING PRIMARY KEY, value DOUBLE);
+
+DESC TABLE test_pk;
+
+DROP TABLE test_pk;
+
+CREATE TABLE test_multiple_pk_definitions (timestamp BIGINT TIME INDEX, host STRING PRIMARY KEY, value DOUBLE, PRIMARY KEY(host));
```

Comment:
What if
```sql
CREATE TABLE test_multiple_pk_definitions (timestamp BIGINT TIME INDEX, host STRING PRIMARY KEY, value DOUBLE, PRIMARY KEY(host) PRIMARY KEY(host));
```

Reply:
It will also throw the duplicate primary key exception. Only `Primary(AKey, BKey, CKey)` is allowed for the system table creation